packages/spec_cli/test/spec_cli_test.dart

```diff
@@ -171,16 +171,17 @@ void main() {
           options: SpecOptions.fromArgs(const ['.', '../b/test', '--no-ci']),
         );
 
+        final lastFrame = testRenderer.frames.last;
+        expect(lastFrame, contains('PASS  ../b/test/another_test.dart'));
+        expect(lastFrame, contains('PASS  test/my_test.dart'));
```

Comment:
I'm not super fond of this approach
It doesn't test the absence of anything extra

For example the output could be:

```dart
 PASS  ../b/test/another_test.dart
 PASS  test/my_test.dart
 PASS  test/my_test.dart
 PASS  test/my_test.dart
 PASS  test/my_test.dart

Test Suites: 2 passed, 2 total
Tests:       2 passed, 2 total
Time:        00:00:00
```

which isn't good

Reply:
If we do something permutations of `anyOf` then, where we include all the strings?

Reply:
Or maybe even simpler, that we verify that the number of lines in the output also is correct?

Reply:
Alright, go ahead! But maybe keep this PR as a backup then, so that the pipeline isn't broken for yet another year? :sweat_smile: